mongodb/src/main/java/org/apache/zeppelin/mongodb/MongoDbInterpreter.java

```diff
@@ -73,7 +73,15 @@ public void open() {
     }
     commandTimeout = Long.parseLong(getProperty("mongo.shell.command.timeout"));
     maxConcurrency = Integer.parseInt(getProperty("mongo.interpreter.concurrency.max"));
-    dbAddress = getProperty("mongo.server.host") + ":" + getProperty("mongo.server.port");
+
+		/* adding support for protocal like mongodb+srv for atlas cluster*/
+    String mongoProtocol = getProperty("mongo.server.protocol", "mongodb");
+		if ("mongodb".equalsIgnoreCase(mongoProtocol)){
+      dbAddress = getProperty("mongo.server.host") + ":" + getProperty("mongo.server.port");
+		}else{
+	    dbAddress = mongoProtocol +"://"+ getProperty("mongo.server.host");
```

Comment:
If we have a protocol value, does it mean `mongo.server.port`? Otherwise, don't some protocols need port value? Could you please verify it?

Reply:
if protocol is ```mongodb``` then it will use ```mongo.server.port``` as many provider use different port for mongo / documentDb or similar service.

but i never seen ```mongodb+srv``` protocol is been used with different port.

Reply:
Ok will update the code as requested.